packages/popover/src/Overlay.js

```diff
@@ -33,6 +32,7 @@ const Overlay = ({
       popoverOpen={popoverOpen}
       overlayOpacity={overlayOpacity}
       zIndex={zIndex}
+      bg={'background.darkest'}
```

Comment:
@wesleymartin85 do we have a replacement for `theme.colors.black` in the palette? `background.darkest` is close, but not identical.